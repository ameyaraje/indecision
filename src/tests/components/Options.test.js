import React from 'react';
import { shallow } from 'enzyme';

import Options from '../../components/Options';

test('should render Options component with no options passed', () => {
    const wrapper = shallow(<Options options={[]} />);

    expect(wrapper).toMatchSnapshot();
});


test('should render Options component with some options passed', () => {
    const wrapper = shallow(<Options options={["option 1", "option 2"]} />);

    expect(wrapper).toMatchSnapshot();
});