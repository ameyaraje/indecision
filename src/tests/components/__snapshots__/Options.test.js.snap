// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Options component with no options passed 1`] = `
<div>
  <div
    className="widget-header"
  >
    <h3
      className="widget-header__title"
    >
      Your options
    </h3>
    <button
      className="button button--link"
    >
      Remove All
    </button>
  </div>
  <p
    className="widget__message"
  >
    Please add an option to get started
  </p>
</div>
`;

exports[`should render Options component with some options passed 1`] = `
<div>
  <div
    className="widget-header"
  >
    <h3
      className="widget-header__title"
    >
      Your options
    </h3>
    <button
      className="button button--link"
    >
      Remove All
    </button>
  </div>
  <Option
    key="option 1"
    optionText="option 1"
  />
  <Option
    key="option 2"
    optionText="option 2"
  />
</div>
`;
